packages/api/tests/endpoints/test-pdrs.js

```diff
@@ -241,3 +289,93 @@ test('DELETE handles the case where the PDR exists in DynamoDb but not in S3', a
   const parsedBody = response.body;
   t.is(parsedBody.detail, 'Record deleted');
 });
+
+test('DELETE removes a PDR from RDS and DynamoDB', async (t) => {
+  // Create the same PDR in Dynamo and PG
+  const newDynamoPdr = fakePdrFactory('completed');
+  const pdrName = newDynamoPdr.pdrName;
+  const newPGPdr = fakePdrRecordFactory({
+    name: pdrName,
+    status: 'completed',
+    collection_cumulus_id: t.context.collectionCumulusId,
+    provider_cumulus_id: t.context.providerCumulusId,
+  });
+
+  // create a new PDR in Dynamo
+  await pdrModel.create(newDynamoPdr);
+
+  // create a new PDR in RDS
+  await pdrPgModel.create(t.context.knex, newPGPdr);
+
+  const response = await request(app)
+    .delete(`/pdrs/${pdrName}`)
+    .set('Accept', 'application/json')
+    .set('Authorization', `Bearer ${jwtAuthToken}`);
+  t.is(response.status, 200);
+
+  // Check Dynamo and RDS. The PDR should have been removed from both.
+  await t.throwsAsync(
+    pdrModel.get({ pdrName }),
+    { instanceOf: RecordDoesNotExist }
+  );
+
+  t.is(
+    await pdrPgModel.exists(t.context.knex, { name: pdrName }),
+    false
+  );
+});
+
+test('DELETE removes a PDR from RDS only if no DynamoDB match exists', async (t) => {
```

Comment:
Ah, this might squash my earlier comment about making it a transaction. So the deletes need to be totally independent? The logic here as I understand it is: 

"Delete the PDR from Dynamo ***and/or*** RDS."

I think that's a little different from what I'm doing with Granules. In that case it's complicated because of files but the way that's written now if the Dynamo record fails to delete, the the PG delete will be rolled back**.

**I'm realizing I don't have a test for this exact case so I could be wrong in what the Granule logic is doing. This is just what was intended.

Reply:
Yeah, granules seem more complex because there are dependent records. I'm still not sure there isn't one direction or another where it should be a transaction. Let's discuss in the architecture meeting.

Reply:
I commented above but wanted to point out here that I don't think a DynamoDB record could _fail_ to delete since it's idempotent. And, similarly, a PG delete that is unsuccessful returns 0 when it fails, it doesn't error. We could update delete in our base model to throw an error though.

Reply:
@botanical I think we wouldn't want to have our delete logic throw errors for delete requests where the record no longer exists, precisely because then a delete would no longer be idempotent. 

Practically, that means a delete query to PG means the record either gets deleted or already was. Either way, the behavior of delete has been satisfied. 

You could still get an error thrown from the PG query due to deadlocking, database unavailability, or any a number of other reasons.

Reply:
@nemreid I think this test and the next one are superfluous? They are proving basically that a DELETE request still succeeds (returns 200) and that the records in both systems definitely don't exist even if one or the other record (Dynamo or PG) already didn't exist.

However, the verification that the delete method succeeds whether the record exists or not seems like it should be at the unit test level for the model `delete` methods if it's not already. And then all we need to verify here is that a DELETE endpoint request deletes records from both systems, which the test above verifies

Reply:
And I don't even think we need unit tests of the  model delete methods to prove that they succeed whether the record exists or not, because that is behavior of the database engines themselves, not our code